pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
+line-length = 120
+target-version = ['py38']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.tox
+    | \.env
+    | \.venv
+    | build
+    | dist
+  )/
+  | scratch.py           # also separately exclude a file named foo.py in
```

Comment:
??

Reply:
In case `black` is applied to whole directory tree with `black .`
`scratch.py` is for local scribbling and testing out. I can remove it if looks surplus.

Reply:
It's just weird to mention a file that is not in the repo, but OK, the rest is also like that.

Reply:
:+1: Removed surplus comment